core/tests/protractor_desktop/topicAndStoryViewer.js

```diff
@@ -54,12 +54,20 @@ describe('Topic and Story viewer functionality', function() {
       objective: 'The goal is to check story viewer functionality.',
       language: 'English'
     };
+
+    var exitTutorial = false;
     for (var i = 1; i <= 3; i++) {
+      if (i === 1) {
+        exitTutorial = true;
+      } else {
+        exitTutorial = false;
+      }
```

Comment:
This can be simplified like I mentioned in the previous comment.

Reply:
Done!